plugin/src/main/java/com/denizenscript/denizen/scripts/commands/item/InventoryCommand.java

```diff
@@ -182,89 +189,10 @@ public void addCustomTabCompletions(TabCompletionsBuilder tab) {
         }
     }
 
-    @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        boolean isAdjust = false, isFlag = false;
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("origin")
-                    && arg.matchesPrefix("origin", "o", "source", "items", "item", "i", "from", "f")
-                    && (arg.matchesArgumentTypes(InventoryTag.class, EntityTag.class, LocationTag.class, MapTag.class)
-                    || arg.matchesArgumentList(ItemTag.class))) {
-                scriptEntry.addObject("origin", Conversion.getInventory(arg, scriptEntry));
-            }
-            else if (!scriptEntry.hasObject("destination")
-                    && arg.matchesPrefix("destination", "dest", "d", "target", "to", "t")
-                    && arg.matchesArgumentTypes(InventoryTag.class, EntityTag.class, LocationTag.class)) {
-                scriptEntry.addObject("destination", Conversion.getInventory(arg, scriptEntry));
-            }
-            else if (!scriptEntry.hasObject("slot")
-                    && arg.matchesPrefix("slot", "s")) {
-                scriptEntry.addObject("slot", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("actions")
-                    && arg.matchesEnumList(Action.class)) {
-                scriptEntry.addObject("actions", arg.asType(ListTag.class).filter(Action.values()));

```

Comment:
... This is unrelated to the PR, but, the fact that this is a list is not documented and is something I've entirely forgotten until now. I vaguely recall there being an actual intended use case though.